h/static/scripts/directive/test/annotation-test.js

```diff
@@ -263,6 +266,20 @@ describe('annotation', function() {
         true,
         mockThreadFilterController.freeze.lastCall.calledWithExactly(false));
     });
+
+    it('does not collapse parent thread after edit', function() {
+      mockAnnotationController.editing.returns(true);
+      link(scope, mockElement, mockAttributes, mockControllers);
+      scope.$digest();
+
+      mockAnnotationController.editing.returns(false);
+      scope.$digest();
+
+      assert.equal(
+        true,
+        mockThreadController.parent.toggleCollapsed.lastCall.calledWithExactly(false));
```

Comment:
You can simplify this slightly to `assert.calledWith(mockThreadController.parent.toggleCollapsed, false)`